### ISSUE TITLE
Component tab - add content dynamically broken HTML

### ISSUE THREAD

**Issue by John-Henrique** [quark-dev/Phonon-Framework#50]
I am trying add lists into content of component tab dynamically but every time broke HTML of tab content. I think has a problem with DOM parser, don`t know how refresh content.

Try with latest version of phonon and simply use

 ```` javascript 
$( 'element' ).html( '<p>my new content</p>' ) 
````

**Comment by Qathom** [comment]
I think you forgot a dot before element, am I right? :)
```
$( '.element' ).html( '<p>my new content</p>' )
```

**Comment by John-Henrique** [comment]
hehehe that's only example...

see this and imagine all HTML necessary (here put only important to case)..

````html
	<nav class="tabs header-tabs">
		<div class="tab-items abas home">
		</div>
		<div class="tab-indicator"></div>
	</nav>
	
	<div class="content conteudo-abas ofertas" data-tab-contents="true" data-disable-swipe="false">
		
	</div>
````
	

Imagine this in loop...
````javascript
// tabs dynamically in loop
tabs  = '<a class="tab-item">Tab 1</a>';
tabs += '<a class="tab-item">Tab 2</a>';
tabs += '<a class="tab-item">Tab 3</a>';
// end loop

$( 'div.abas' ).html( tabs );


// imagine other loop again
html  = '<div class="tab-content"><ul class="list" ><li>Item 1</li><li>Item 2</li><li>Item 3</li></ul></div>';
html += '<div class="tab-content"><ul class="list" ><li>Item 1</li><li>Item 2</li><li>Item 3</li></ul></div>';
html += '<div class="tab-content"><ul class="list" ><li>Item 1</li><li>Item 2</li><li>Item 3</li></ul></div>';
// end loop

$( '.conteudo-abas' ).html( html );
````

This creates a broken list when done using JavaScript, but work fine (with same HTML produced) if put directly in HTML page.

**Comment by Qathom** [comment]
Thank you for your example.
Yes, this is not possible because the tabs are already "defined and styled".
I'll think about this to make it possible asap.

**Comment by John-Henrique** [comment]
Fixed in [v1.2.0](https://github.com/quark-dev/Phonon-Framework/releases/tag/v1.2.0)
